XRFoundation/XREngine
# Make MaterialLibrary Stateful

* fix state issues in Material Library Panel
to test:
checkout branch, open a scene and try selecting + modifying materials in the material library. The properties panel should now correctly update when you change the selected material
